modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceAccelerators.path

```diff
@@ -103,6 +103,11 @@
 	<td>//div[contains(@class,'price')]//span[contains(@class,'price-value-discount')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PRODUCT_DETAILS_VALUE_INACTIVE_PROMO_PRICE</td>
+	<td>//p[@title='${key_productName}']//following-sibling::p[@class='card-text']//span[@class='price-value price-value-inactive']//following-sibling::span[@class='price-value price-value-promo']</td>
+	<td></td>
+</tr>
```

Comment:
This locator is too complex, please try to find and use a path that identifies just the promo price in a product card. Also, please move this locator to `CommerceFrontStore.path`.
```suggestion
<tr>
	<td>PRODUCT_CARD_PROMO_PRICE</td>
	<td>//p[@title='${key_productName}']//following-sibling::p[@class='card-text']//span[contains(@class,'price-value-promo')]</td>
	<td></td>
</tr>
```